### PR TITLE
Fix proxies when using Google authentication

### PR DESCRIPTION
**Pull request by tboyce1**
Fixes https://github.com/PokemonGoMap/PokemonGo-Map/issues/754

### CODE REVIEW

**Comment by tboyce1** [comment]
I have no idea what this actually does, but it fixes the issue.

File: pgoapi/auth_google.py [hunk 1]
```diff
@@ -44,6 +45,7 @@ def __init__(self):
 
         self._auth_provider = 'google'
 
+        self._session = requests.session()
```
Comment (tboyce1) [inline pgoapi/auth_google.py]: I have no idea what this actually does, but it fixes the issue.

**Comment by Nik0lashka** [comment]
@invisiblek @chuyskywalker could you please review this PR?

**Comment by invisiblek** [comment]
This isn't going to work, we're not using requests to make the call to google (we use gpsoauth, which in turn uses requests, but doesn't support proxies). Sure it fixes it so the thing doesn't crash, but its not actually using the proxy. We should just remove the change here unless someone wants to submit a PR to https://github.com/simon-weber/gpsoauth

**Comment by invisiblek** [comment]
https://github.com/PokemonGoMap/pgoapi/pull/2

**Comment by Nik0lashka** [comment]
@invisiblek just to clarify - if i want use fully proxy connection to evade IP ban, i better create PTC account instead of authenticating trough google and use proxy setting then?

**Comment by invisiblek** [comment]
Correct. Or, like I said, get a patch into the gpsoauth project that adds proxy support, then we'd be able to support it.

**Comment by invisiblek** [comment]
I created a pull request for gpsoauth to support proxies. If it gets merged I have a patch for pgoapi to support it. For the time being we should probably just remove the changes I made to auth_google.py so things work.

**Comment by invisiblek** [comment]
Well, that escalated quickly. Patch merged into gpsoauth already

**Pull request closed without merging**